Gifski/Gifski.swift

```diff
@@ -291,22 +291,35 @@ final class Gifski {
 						return
 					}
 
-					let image = result.image
+					if !isEstimation, result.completedCount == 1 {
+						Crashlytics.record(
+							key: "\(debugKey): CGImage",
+							value: result.image.debugInfo
+						)
+
+						// TODO: Remove
+						print(result.image.debugInfo)
+						DispatchQueue.main.async {
+							NSAlert.showModal(title: "Debug info:", message: result.image.debugInfo)
+						}
+					}
 
-					guard let bytePointer = image.bytePointer else {
-						completionHandlerOnce(.failure(.generateFrameFailed(
-							NSError.appError("Could not get byte pointer of image data provider.")
-						)))
+					let pixels: CGImage.Pixels
+					do {
+						pixels = try result.image.pixels(as: .argb, premultiplyAlpha: false)
```

Comment:
Is there a reason why you use `argb` instead of `rgba`? RGBA is gifski's native format, so this will make it add an extra conversion step to get RGBA.

Reply:
@kornelski So it turns out `vImage` adds padding to the image pixel rows. That's why there was this bug. Is there any reason `gifski_add_frame_rgba` doesn't have a `bytesPerRow` argument? Because that's required to be able to switch to `.rgba`.

Reply:
I see. Lack of bytes per row was just an omission for the rgba call. I guess in that case it's fine if you continue with argb.